sig-security-charter.md

```diff
@@ -0,0 +1,64 @@
+# SIG Security Charter
+
+## Scope
+
+SIG Security covers horizontal security initiatives for the OpenTelemetry (OTel)
+project and is focused on improving the security of the project across all
+components.
+
+This includes preparing the project for regular security audits, improving the
+vulnerability management process, producing cross-cutting security
+documentation, serving as a security community touch point, managing the
+non-emnargoed (public) vulnerability process, default repo configurations, and
+defining the bug bounty.
+
+As a process-oriented SIG, SIG Security does not directly own OpenTelemetry component
+code.
+
+### In scope
+
+#### Vulnerability Management Process
+
+Work with the OpenTelemetry Technical Committee (TC) to define the processes for fixing and disclosing vulnerabilities. For example:
+
+- The default level for OTel repo Github security configuration and tooling
+- When the private fix & release process is invoked
+- How vulnerabilities are rated
+- The scope of the bug bounty
+- Post-announcement follow-ups, such as additional fixes, mitigations, preventions or documentation after a vulnerability is made public
+- Distributor announcement policies, such as timelines, criteria for joining the list, etc.
+- How, when and where vulnerabilities are announced
+
+#### Security Community Management and Outreach
+
+Provide an entry point to the OTel community for new security-minded contributors, as well as a meeting point to discuss security themes and issues within OTel, including:
+
+- Answer security questions from inexperienced users (that don't know what SIG to go to), and identify common questions or issues as areas for improvement.
+- Provide an "entry point" for new contributors interested in security. Route these new contributors to other SIGs when they have more specific goals (e.g. SIG Node for container isolation).
```

Comment:
nit: some lines are being wrapped, other are not. While I prefer to not wrap the lines automatically (I can have my code editor do that for me according to my taste), I don't mind forcing wraps, as long as it's consistent across the whole document :-)

Reply:
Habit from working with Reiley's team haha. Updated